diff --git a/include/general/arg_main.h b/include/general/arg_main.h
--- a/include/general/arg_main.h
+++ b/include/general/arg_main.h
@@ -434,6 +434,7 @@
 int TestMTD_main(int argc,char ** argv);
 int TestCmds_main(int argc,char ** argv);
 int Apero2PMVS_main(int argc,char ** argv);
+int Apero2MeshlabRast_main(int argc,char ** argv);
 int Ori2XML_main(int argc,char ** argv);
 
 // uti_images
diff --git a/src/CBinaires/CMakeLists.txt b/src/CBinaires/CMakeLists.txt
--- a/src/CBinaires/CMakeLists.txt
+++ b/src/CBinaires/CMakeLists.txt
@@ -40,6 +40,9 @@
 	add_executable(Apero2PMVS Apero2PMVS.c)
 	INSTALL(TARGETS Apero2PMVS RUNTIME DESTINATION ${Install_Dir})
 
+    add_executable(Apero2Meshlab Apero2Meshlab.c)
+    INSTALL(TARGETS Apero2Meshlab RUNTIME DESTINATION ${Install_Dir})
+
 	add_executable(mmxv mmxv.c)
 	INSTALL(TARGETS mmxv RUNTIME DESTINATION ${Install_Dir})
 
diff --git a/src/CBinaires/mm3d.cpp b/src/CBinaires/mm3d.cpp
--- a/src/CBinaires/mm3d.cpp
+++ b/src/CBinaires/mm3d.cpp
@@ -167,6 +167,7 @@
        aRes.push_back(cMMCom("Digeo",Digeo_main," In devlopment- Will compute tie points "));
        aRes.push_back(cMMCom("AperoChImSecMM",AperoChImMM_main,"Select secondary images for MicMac",cArgLogCom(2)));
 	   aRes.push_back(cMMCom("Apero2PMVS",Apero2PMVS_main,"Convert Orientation from Apero-Micmac workflow to PMVS format"));
+       aRes.push_back(cMMCom("Apero2Meshlab",Apero2MeshlabRast_main,"Convert Orientation from Apero-Micmac workflow to a meshlab-compatible format"));
        aRes.push_back(cMMCom("Bascule",Bascule_main," Generate orientations coherent with some physical information on the scene",cArgLogCom(2)));
        aRes.push_back(cMMCom("BatchFDC",BatchFDC_main," Tool for batching a set of commands"));
        aRes.push_back(cMMCom("Campari",Campari_main," Interface to Apero, for compensation of heterogenous measures",cArgLogCom(2)));
diff --git a/src/uti_files/Sources.cmake b/src/uti_files/Sources.cmake
--- a/src/uti_files/Sources.cmake
+++ b/src/uti_files/Sources.cmake
@@ -1,14 +1,15 @@
-set(uti_files_Src_Files
-    ${UTI_FILES_DIR}/CPP_MapCmd.cpp
+set(uti_files_Src_Files
+    ${UTI_FILES_DIR}/CPP_MapCmd.cpp
     ${UTI_FILES_DIR}/CPP_BatchFDC.cpp
     ${UTI_FILES_DIR}/CPP_MyRename.cpp
     ${UTI_FILES_DIR}/CPP_cod.cpp
     ${UTI_FILES_DIR}/CPP_GGP_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_Ori_txt2Xml.cpp
     ${UTI_FILES_DIR}/CPP_TestKeys.cpp
-    ${UTI_FILES_DIR}/CPP_TestMTD.cpp
-    ${UTI_FILES_DIR}/CPP_TestCmds.cpp
+    ${UTI_FILES_DIR}/CPP_TestMTD.cpp
+    ${UTI_FILES_DIR}/CPP_TestCmds.cpp
     ${UTI_FILES_DIR}/CPP_Apero2PMVS.cpp
+    ${UTI_FILES_DIR}/CPP_Apero2Meshlab.cpp
     ${UTI_FILES_DIR}/CPP_GenHeadTifTile.cpp
     ${UTI_FILES_DIR}/CPP_Ori2Xml.cpp
     ${UTI_FILES_DIR}/CPP_Prep4masq.cpp
