diff -r 6c9cd33045a1 include/general/arg_main.h
--- a/include/general/arg_main.h	Thu Nov 07 17:43:35 2013 +0100
+++ b/include/general/arg_main.h	Fri Nov 08 10:34:04 2013 +0100
@@ -434,6 +434,7 @@
 int TestMTD_main(int argc,char ** argv);
 int TestCmds_main(int argc,char ** argv);
 int Apero2PMVS_main(int argc,char ** argv);
+int Apero2Meshlab_main(int argc,char ** argv);
 int Ori2XML_main(int argc,char ** argv);
 int GenCode_main(int argc,char ** argv);
 int XifGps2Xml_main(int argc,char ** argv);
diff -r 6c9cd33045a1 src/CBinaires/CMakeLists.txt
--- a/src/CBinaires/CMakeLists.txt	Thu Nov 07 17:43:35 2013 +0100
+++ b/src/CBinaires/CMakeLists.txt	Fri Nov 08 10:34:04 2013 +0100
@@ -60,6 +60,9 @@
 	add_executable(Apero2PMVS Apero2PMVS.c)
 	INSTALL(TARGETS Apero2PMVS RUNTIME DESTINATION ${Install_Dir})
 
+    add_executable(Apero2Meshlab Apero2Meshlab.c)
+    INSTALL(TARGETS Apero2Meshlab RUNTIME DESTINATION ${Install_Dir})
+
 	add_executable(mmxv mmxv.c)
 	INSTALL(TARGETS mmxv RUNTIME DESTINATION ${Install_Dir})
 
diff -r 6c9cd33045a1 src/CBinaires/mm3d.cpp
--- a/src/CBinaires/mm3d.cpp	Thu Nov 07 17:43:35 2013 +0100
+++ b/src/CBinaires/mm3d.cpp	Fri Nov 08 10:34:04 2013 +0100
@@ -168,6 +168,7 @@
        aRes.push_back(cMMCom("Digeo",Digeo_main," In devlopment- Will compute tie points "));
        aRes.push_back(cMMCom("AperoChImSecMM",AperoChImMM_main,"Select secondary images for MicMac",cArgLogCom(2)));
 	   aRes.push_back(cMMCom("Apero2PMVS",Apero2PMVS_main,"Convert Orientation from Apero-Micmac workflow to PMVS format"));
+       aRes.push_back(cMMCom("Apero2Meshlab",Apero2Meshlab_main,"Convert Orientation from Apero-Micmac workflow to a meshlab-compatible format"));
        aRes.push_back(cMMCom("Bascule",Bascule_main," Generate orientations coherent with some physical information on the scene",cArgLogCom(2)));
        aRes.push_back(cMMCom("BatchFDC",BatchFDC_main," Tool for batching a set of commands"));
        aRes.push_back(cMMCom("Campari",Campari_main," Interface to Apero, for compensation of heterogenous measures",cArgLogCom(2)));
diff -r 6c9cd33045a1 src/uti_files/Sources.cmake
--- a/src/uti_files/Sources.cmake	Thu Nov 07 17:43:35 2013 +0100
+++ b/src/uti_files/Sources.cmake	Fri Nov 08 10:34:04 2013 +0100
@@ -1,24 +1,25 @@
 set(uti_files_Src_Files
     ${UTI_FILES_DIR}/CPP_XifGps2Xml.cpp
     ${UTI_FILES_DIR}/CPP_MapCmd.cpp
-    ${UTI_FILES_DIR}/CPP_BatchFDC.cpp
-    ${UTI_FILES_DIR}/CPP_MyRename.cpp
-    ${UTI_FILES_DIR}/CPP_cod.cpp
-    ${UTI_FILES_DIR}/CPP_GGP_txt2Xml.cpp
-    ${UTI_FILES_DIR}/CPP_Ori_txt2Xml.cpp
-    ${UTI_FILES_DIR}/CPP_TestKeys.cpp
+    ${UTI_FILES_DIR}/CPP_BatchFDC.cpp
+    ${UTI_FILES_DIR}/CPP_MyRename.cpp
+    ${UTI_FILES_DIR}/CPP_cod.cpp
+    ${UTI_FILES_DIR}/CPP_GGP_txt2Xml.cpp
+    ${UTI_FILES_DIR}/CPP_Ori_txt2Xml.cpp
+    ${UTI_FILES_DIR}/CPP_TestKeys.cpp
     ${UTI_FILES_DIR}/CPP_TestMTD.cpp
     ${UTI_FILES_DIR}/CPP_TestCmds.cpp
-    ${UTI_FILES_DIR}/CPP_Apero2PMVS.cpp
-    ${UTI_FILES_DIR}/CPP_GenHeadTifTile.cpp
-    ${UTI_FILES_DIR}/CPP_Ori2Xml.cpp
-    ${UTI_FILES_DIR}/CPP_Prep4masq.cpp
-    ${UTI_FILES_DIR}/CPP_GenCode.cpp
-)
-
-SOURCE_GROUP(uti_files FILES ${uti_files_Src_Files})
-
-set(Elise_Src_Files
-	${Elise_Src_Files}
-	${uti_files_Src_Files}
-)
+    ${UTI_FILES_DIR}/CPP_Apero2PMVS.cpp
+    ${UTI_FILES_DIR}/CPP_Apero2Meshlab.cpp
+    ${UTI_FILES_DIR}/CPP_GenHeadTifTile.cpp
+    ${UTI_FILES_DIR}/CPP_Ori2Xml.cpp
+    ${UTI_FILES_DIR}/CPP_Prep4masq.cpp
+    ${UTI_FILES_DIR}/CPP_GenCode.cpp
+)
+
+SOURCE_GROUP(uti_files FILES ${uti_files_Src_Files})
+
+set(Elise_Src_Files
+	${Elise_Src_Files}
+	${uti_files_Src_Files}
+)
